libs/openFrameworks/math/ofQuaternion.h

```diff
@@ -199,21 +199,21 @@ ofQuaternion::ofQuaternion() {
 
 
 //----------------------------------------
-ostream& ofQuaternion::operator<<(ostream& os) {
-    os << _v.x << ", " << _v.y << ", " << _v.z << ", " << _v.w;
+ostream& operator<<(ostream& os, const ofQuaternion &q) {
```

Comment:
this `operator<<` is not class method.
i defined this as friend function in `ofQuaternion`.

maybe cause is not this. maybe...